lib/alchemy/admin/locale.rb

```diff
@@ -46,15 +46,16 @@ def available_locale
       def locale_from_user
         return if !current_alchemy_user
         if user_has_preferred_language?
-          current_alchemy_user.language
+          current_alchemy_user.language.to_sym
```

Comment:
Because of the `set_translation` method. https://github.com/IngoAlbers/alchemy_cms/blob/141b6edc9f75e0cc6572aaa67e485e49442cc754/lib/alchemy/admin/locale.rb#L18. We detect by `to_sym` but we just set the actual locale then, which would be the model instance in my case. I am open to change it in any way that fits best though.